.github/CODEOWNERS

```
@@ -8,20 +8,20 @@
 # directory and file-level owners. Feel free to add to this!
 
 /packages/celotool/ @celo-org/devopsre
-/packages/cli/ @celo-org/applications
-/packages/dev-utils/ @celo-org/applications
-/packages/docs/ @celo-org/applications @celo-org/devrel
-/packages/env-tests/ @celo-org/applications
-/packages/faucet/ @celo-org/applications
+/packages/cli/ @celo-org/devtooling
+/packages/dev-utils/ @celo-org/devtooling
+/packages/docs/ @celo-org/devtooling @celo-org/devrel
+/packages/env-tests/ @celo-org/devtooling
```

```suggestion
/packages/env-tests/ @celo-org/primitives
```

I suggest env-test is owned by the primitive team.
See this [Slack](https://celo-org.slack.com/archives/C9RSRC8KS/p1696860716280309?thread_ts=1696860617.336019&cid=C9RSRC8KS) thread where we asked what it's about.

---

agree!